offline/p300_paradigm.py

```diff
@@ -13,17 +13,26 @@
 
 from pylsl import local_clock, StreamInfo, StreamOutlet
 
+# Load the configurations of the paradigm
+with open(r'configs/config_long_red-yellow.txt', 'rb') as file:
+    params = pickle.load(file)
+
 # Define custom colors
-GREY = (103, 103, 110)
+GREY = (0, 0, 0)
 WHITE = (255, 255, 255)
-RED = (255, 0, 0)
-YELLOW = (255, 255, 0)
 
-WINDOW_WIDTH = 600 # 600
-WINDOW_HEIGHT = 600 # 600
-# Load the configurations of the paradigm
-with open(r'./config_durations.pkl', 'rb') as file:
-    exp_durations = pickle.load(file)
+TARGET = params['target_color']
+HIGHLIGHT = params['highlight_color']
+
+# Initialise Pygame
+pg.init()
+
+# Set the window size parameters
+sc = pg.display.Info()
+WIDTH = sc.current_w
+HEIGHT = sc.current_h
```

Comment:
I would like to move the Pygame initialisation back to its original location.
Since they have become runtime constants, I do not think it makes any sense to leave them as constants here. They were originally here because it provides a better overview, but they are only actually used once within the `__init__()` call. Since they are now defined on runtime, I think it makes more sense to move it there.

I also just found this: https://stackoverflow.com/a/65706030 which might be worth considering for the other constants.